guides/v1.10.0/cookbook/client_server_interaction/serving_compiled_templates_using_nodejs.md

```diff
@@ -138,4 +138,4 @@ tutorial is based on is found at http://www.github.com/stevekane/precompiled-tem
 As a bonus, if you are interested in using ```Emblem.js``` which has a HAML-like syntax to 
 build your templates, there is a grunt task for precompiling emblem.js templates called 
 ```grunt-emblem```.  The steps needed to configure this task are similar to the ones shown
-but be sure to refer to the projects github repository for details.
+but be sure to refer to the projects Github repository for details.
```

Comment:
```suggestion
but be sure to refer to the projects GitHub repository for details.
```

The capitalization should be GitHub. This will need to be changed in multiple places.

Reply:
Fixed Github -> GitHub